### PR TITLE
update vendor deps

### PR DESCRIPTION
**Pull request by xh3b4sd**

### CODE REVIEW

File: integration/key/release_name.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package key
+
+import (
+	"fmt"
+)
+
+func AWSConfigReleaseName(clusterID string) string {
```
Comment (rossf7) [inline integration/key/release_name.go]: Shouldn't this be for Azure?

File: integration/key/release_name.go [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+	return fmt.Sprintf("e2esetup-awsconfig-%s", clusterID)
```
Comment (MarcelMue) [inline integration/key/release_name.go]: Seems like a mistake

File: integration/key/release_name.go (hunk 1 above)
Comment (xh3b4sd) [reply integration/key/release_name.go]: I copied the whole file from the `aws-operator`. Dropped the dead code.

File: integration/key/release_name.go (hunk 2 above)
Comment (xh3b4sd) [reply integration/key/release_name.go]: I copied the whole file from the `aws-operator`. Dropped the dead code.

**Pull request merged**